### PR TITLE
Refine French translation

### PR DESCRIPTION
**Pull request by yvesgurcan**
During my playthrough of the first map of the Wizard's Isle campaign, I noticed that several of the translated French strings could be improved.

### CODE REVIEW

**Comment by dimag0g** [comment]
Hi @yvesgurcan, these are nice improvements for the most part. I would nevertheless suggest that we stick to the GoG translation for monster/building names ("esprits" for elementals and "fonderie renfocée"). I also ask myself how "Vitesse ennemi" would look on the interface as "Vitesse CPU" is already a tad too long.

Untitled
Untitled3
Untitled2

**Comment by zenseii** [comment]
>  I also ask myself how "Vitesse ennemi" would look on the interface as "Vitesse CPU" is already a tad too long.

The Vitesse ennemi part can easily and in my opinion should be made into two lines similar to what has been done elsewhere. Other official translations do use two lines for this specific menu when necessary. Obviously text that fit well on a single line should remain right above the icon like itinéraire in the picture above.

File: files/lang/fr.po [hunk 1]
```diff
@@ -3575,7 +3575,7 @@ msgstr ""
 msgid ""
 "One or more heroes may still move, are you sure you want to end your turn?"
 msgstr ""
-"Un ou plusieurs héros peuvent encore bouger, êtes-vous sûr de vouloir "
+"Un ou plusieurs héros peuvent encore se déplacer. Êtes-vous sûr de vouloir "
```
Comment (felix642) [inline files/lang/fr.po:3578]: Do we support the "Ê" letter ?

**Review by felix642 — COMMENTED** [review]
Hi @yvesgurcan, looks good !
I only have one small question. Would you mind taking a look ?

File: files/lang/fr.po (hunk 1 above)
Comment (dimag0g) [reply files/lang/fr.po:3578]: @felix642  We do in the sense that "Ê" gets replaced by "E" before machine translation files are generated. When "Ê" is supported in fheroes2, we will just undo the replacement command and "Ê" will be used in the game for real.

File: files/lang/fr.po [hunk 2]
```diff
@@ -290,7 +290,7 @@ msgid "As your enemy reaches for the %{name}, it mysteriously disappears."
 msgstr "Tandis que votre ennemi atteint %{name}, il disparait mystérieusement."
 
 msgid "You have captured an enemy artifact!"
-msgstr "Vous avez capturé un artéfact de l'ennemi !"
+msgstr "Vous avez capturé un artéfact ennemi !"
```
Comment (FlorianBaba) [inline files/lang/fr.po:293]: There is no accent on this word in french :wink: 
```suggestion
msgstr "Vous avez capturé un artefact ennemi !"
```

File: files/lang/fr.po [hunk 3]
```diff
@@ -2111,7 +2111,7 @@ msgid "Buy Monsters"
 msgstr "Acheter créatures"
 
 msgid "Town Population Information and Statistics"
-msgstr "Information et statistiques sur la population du village"
+msgstr "Information et statistiques sur la population de la ville"
```
Comment (FlorianBaba) [inline files/lang/fr.po:2114]: Information**s** can be plural here, unlike information written in english
```suggestion
msgstr "Informations et statistiques sur la population de la ville"
```

File: files/lang/fr.po [hunk 4]
```diff
@@ -2779,7 +2779,7 @@ msgid "Don't Show"
 msgstr "Masquer"
 
 msgid "Enemy Speed"
-msgstr "Vitesse CPU"
+msgstr "Vitesse ennemis"
```
Comment (FlorianBaba) [inline files/lang/fr.po:2782]:
```suggestion
msgstr "Vitesse des ennemis"
```

File: files/lang/fr.po [hunk 5]
```diff
@@ -4781,7 +4781,7 @@ msgid ""
 "in a fiery pool of molten lava. A group of them approach you and offer their "
 "services. Would you like to recruit Fire Elementals?"
 msgstr ""
-"Sous une structure qui sert à retenir la chaleur, des élémentaires du feu se "
+"Sous une structure qui sert à retenir la chaleur, des élémentaires de feu se "
```
Comment (FlorianBaba) [inline files/lang/fr.po:4784]:
```suggestion
"Dans une mare ardente de lave en fusion, des élémentaires de feu se "
```

**Comment by ihhub** [comment]
We are closing this pull request in favor of #5057 .

**Pull request closed without merging**